site/content/en/docs/Guides/Best Practices/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Cloud Best Practices" 
+linkTitle: "Cloud Best Practices"
+date: 2023-05-12T00:00:00Z
+weight: 9
+description: "Best practices for running Agones in production."
+---
+
+## Overview
+
+Running Agones in production takes consideration, from planning your launch to figuring
+out the best course of action for cluster and Agones upgrades. On this page, we've collected
+some general best practices. We also have cloud specific pages for:
+
+* [Google Kubernetes Engine (GKE)]({{< relref "gke.md" >}})
+
+If you are interested in submitting best practices for your cloud prodiver / on-prem, [please contribute!]({{< relref "/Contribute" >}})
+
+## Separate game servers from other workloads
+
+We recommend using [taints and tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/) to ensure that your game servers are running separate from Agones controllers. There are many ways to accomplish this, here we discuss one:
```

Comment:
This section loses the information from the prior best practice that agones by default is configured to put the system components on these nodes. Also with @gongmax's PR, we could actually suggest setting the helm property to force the system components onto these nodes.

Reply:
Let's make that change when #3161 goes in?